### PR TITLE
improve docker and docker compose sections

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: src/docs/user/references/ScriptLanguageSupport.adoc [hunk 1]
```diff
@@ -398,13 +398,13 @@ my $USERSPACE= $ENV{"USERSPACE"};
 
 === Docker Compose
 
-The execution of a Docker_Compose task requires that Docker and Docker Compose are installed on the host machine of the ProActive Node. Please refer to the official
+The execution of a Docker_Compose task requires the installation of Docker and Docker Compose on the host machine of the ProActive Node. Please refer to the official
```
Comment (mklkun) [inline src/docs/user/references/ScriptLanguageSupport.adoc]: installation of Docker => installation of both Docker

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -132,11 +132,11 @@ link:https://www.perl.org/[Perl, window="_blank"],
 link:https://www.gnu.org/software/bash/[Bash, window="_blank"],
 link:https://en.wikipedia.org/wiki/Unix_shell[Any Unix Shell, window="_blank"],
 link:https://en.wikipedia.org/wiki/Cmd.exe[Windows CMD, window="_blank"],
-link:https://docs.docker.com/engine/reference/builder/[Docker File, window="_blank"],
+link:https://docs.docker.com/engine/reference/builder/[Dockerfile, window="_blank"],
 link:https://docs.docker.com/compose/compose-file/[Docker Compose, window="_blank"] and
 link:https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/[Kubernetes, window="_blank"].
 
-NOTE: Though `Docker File`, `Docker Compose` and `Kubernetes` are not really scripting language but rather description languages, ProActive tasks can interpret Docker files, Docker Compose or Kubernetes yaml files to start containers or pods.
+NOTE: Though `Dockerfile`, `Docker Compose` and `Kubernetes` are not really scripting language but rather description languages, ProActive tasks can interpret Docker files, Docker Compose or Kubernetes yaml files to start containers or pods.
```
Comment (mboussaa) [inline src/docs/user/ProActiveUserGuide.adoc]: what does mean Though `Dockerfile` ?  maybe In contrast to Dockerfiles? 
Docker files => Dockerfiles

File: src/docs/user/references/ScriptLanguageSupport.adoc [hunk 3]
```diff
@@ -437,20 +437,19 @@ docker-compose --verbose up --exit-code-from helloworld
 If splitting by space is prohibitive you can specify the split regex in the <<_glossary_generic_information, generic informations>> with the
 key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value, then your *docker-compose-up-options* will need to look like this: "--option1!SPLIT!--option2".
 
-=== Docker File
+=== Dockerfile
 
-The main behavior of a Docker_File task is to first build an image and then run a container instance from it. Once the execution is done, the container is stopped and the built image is deleted.
+The default behavior of the Dockerfile task is to build an image and then start a container instance from it. Once the execution is done, the container is stopped and the built image is deleted. The build, start, stop and remove actions can be parametrized through advanced command line options which will be detailed aferward.
```
Comment (mboussaa) [inline src/docs/user/references/ScriptLanguageSupport.adoc]: , and remove
parametrized => tuned
aferward => afterward, but I prefer later on than afterward

File: src/docs/user/references/ScriptLanguageSupport.adoc [hunk 4]
```diff
@@ -437,20 +437,19 @@ docker-compose --verbose up --exit-code-from helloworld
...
+The execution of a Dockerfile task requires the installation of Docker engine on the host machine of the ProActive Node. Please refer to the official Docker documentation to see how to install https://docs.docker.com/engine/installation/[Docker^].
 
-To use a Docker_File task, put the content of a Dockerfile inside the _Task Implementation_ section. You can find out how to write Dockerfile
+To use a Dockerfile task, you have to put the content of a Dockerfile inside the _Script_ editor of the *Task Implementation* section. You can find out how to write Dockerfile
 in the official https://docs.docker.com/engine/reference/builder/[Dockerfile documentation^].
-A Docker_File task allows executing a succession of docker commands according to the lifecycle of Docker containers. In order, docker build, docker run, docker stop, and docker rmi are ran when a Docker_File task is executed.
+A Dockerfile task allows executing a succession of docker commands according to the lifecycle of Docker containers.
+//In order, docker build, docker run, docker stop, and docker rmi are ran when a Dockerfile task is executed.
```
Comment (mboussaa) [inline src/docs/user/references/ScriptLanguageSupport.adoc]: this info is important though

**Pull request merged**